local_settings.py.tmpl

```diff
@@ -112,6 +112,12 @@ ALLOWED_DOCUMENT_TYPES = [
 ]
 MAX_DOCUMENT_SIZE = 100
 
+STANDALONE_APPS = (
```

Comment:
Like for old platform, please, do not duplicate this list into `INSTALLED_APPS` but concatenate `STANDALONE_APPS` to it: https://github.com/gem/oq-platform/blob/master/openquakeplatform/openquakeplatform/local_settings.py.template#L292

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/23/commits/96509f853fb9d64aca825ab47c46b30dbff80311